frontend/src/components/page_content/plan/budgeting.js

```diff
@@ -1,39 +1,47 @@
 import React from "react";
-import Center from "react-center";
 import Style from "style-it";
+import ToastToggle from "../../toast/toast-toggle";
+import { numWithCommas, calcMonthlyValue } from "../../../utils/plan-utils";
 
 import { connect } from "react-redux";
 
-import { RadialChart } from "react-vis";
-
 import {
-  getFixedExpenses,
   getSavings,
+  getFixedExpenses,
   getSpending,
 } from "../../../redux/selectors";
 
-import ToastToggle from "../../toast/toast-toggle";
+import ToastPieChart from "../../toast/toast-pie-chart";
 
 class Budgeting extends React.Component {
   constructor(props) {
     super(props);
 
-    this.budgeting = [
+    this.data = [
       {
-        angle: this.props.fixedExpenses,
-        label: "$" + this.props.fixedExpenses + " fixed expenses monthly",
+        name: "Spending",
+        value: this.props.spending,
+        fill: "var(--toast-purple-2)",
       },
       {
-        angle: this.props.spending,
-        label: "$" + this.props.spending + " spending monthly",
+        name: "Fixed Expenses",
+        value: this.props.fixedExpenses,
+        fill: "var(--toast-purple-3)",
       },
       {
-        angle: this.props.savings,
-        label: "$" + this.props.savings + " savings monthly",
+        name: "Savings",
+        value: this.props.savings,
+        fill: "var(--toast-purple-1)",
       },
     ];
   }
 
+  getCaption() {
+    return `Given that your monthly income is $${numWithCommas(
+      calcMonthlyValue(this.props.salaryAfterTax)
+    )}, we recommend you set aside $3,625 for fixed expenses or expenses that can’t be easily changed such as rent/mortgage or subscriptions. We also recommend you set aside $2,175 for spending and $1,450 for savings for this month.  `;
```

Comment:
you should fill in the number values with variables unless we would have to add a util for it (backend will be doing it so we don't wanna make more plan utils to throw them away)